utils/node_local_test.py

```diff
@@ -3125,7 +3125,8 @@ def set_server_fi(server):
     cmd_env = get_base_env()
 
     cmd_env['OFI_INTERFACE'] = 'eth0'
-    cmd_env['CRT_PHY_ADDR_STR'] = 'ofi+sockets'
+    cmd_env['CRT_PHY_ADDR_STR'] = 'ofi+tcp;ofi_rxm'
```

Comment:
I've got a PR to pull this from the nlt_server.yaml file, I'm not sure why you've used two different values in the two locations though.

This is used for the cart binaries which don't get the information from the agent so I'd expect it to match what the rest of the testing is using unless there's a good reason.

Reply:
On the other hand if this works then I'd be happy for you to land it, and when my PR to load this from the config file lands then it'll be consistent again.